Bottom: 90b92c8781bd2dd70ec37397cba339806d281a46
Top:    180d8f2c9a22f1039d8ff0813ee06dacda16eb3c
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-09 15:54:58 -0500

Update ptxdistrc filename


---
diff --git a/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git b/docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
similarity index 100%
rename from docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-git
rename to docker/buildbot-worker-ptxdist/buildbot/.ptxdist/ptxdistrc-2012-09
